ci/jobs/pulp-upgrade.yaml

```diff
@@ -167,12 +167,44 @@
             pulp-admin rpm repo list --repo-id rhel7-rhn-tools
 
             pulp-admin logout
+        - shell:
+            echo "BASE_URL=https://$(hostname --long)" > parameters.txt
+            cp /etc/pki/CA/cacert.pem cacert.pem
+        - inject:
+            properties-file: parameters.txt
+        - trigger-builds:
+            - project:
+                - pulp-smash-runner
+              block: true
+              predefined-parameters: |
+                  BASE_URL=$BASE_URL
+                  PULP_VERSION=$UPGRADE_PULP_VERSION
```

Comment:
Is there a reason that `BASE_URL` is injected into the current environment via the `parameters.txt` file, but not `UPGRADE_PULP_VERSION`?